Assets/Scripts/UI/Components/ATM/Timeline_Regulator.cs

```diff
@@ -22,7 +22,7 @@ public class Timeline_Regulator : MonoBehaviour
 
         private List<RectTransform> yearInstances = new List<RectTransform>();
         public int currentYear;
-        public UnityEvent<int> YearChanged;
+        public UnityEvent<int> YearChanged = new ();
```

Comment:
misschien dit event gelijk trekken met de atmdatacontroller?

Reply:
of andersom :)

Reply:
Dat is geen UnityEvent maar een event, dus die moet niet met new worden geinitialiseerd